AUTHORS

```diff
@@ -181,7 +181,6 @@ Ryan Petrello <lists@ryanpetrello.com>
 Fuji Goro <gfuji@cpan.org>
 Siddharth Mahendraker <siddharth_mahen@hotmail.com>
 Dave Pacheco <dap@joyent.com>
-Mathias Buus <m@ge.tt>
```

Comment:
>  If there are duplicates, then the new entries should be dropped

You can't generalize like that. I'd wager that for most authors, the later entries are more accurate (when they fixed their setup etc), but in the end, you just have to ask them to be sure.